acknowledging.html

```diff
@@ -170,28 +190,32 @@ <h1 id="acknowledging-or-citing-astropy">Acknowledging or Citing Astropy<a class
 
 		<h2>In Publications</h2>
 
-		<p>If you use Astropy for work/research presented in a publication (whether directly, or as a dependency to another package), we ask that you please cite the Astropy papers:
+		<p>If you use Astropy for work/research presented in a publication (whether directly, or as a dependency to another package), we ask that you please cite the three Astropy papers:
             <ul>
-                <li><a href="https://arxiv.org/abs/1801.02634" target="_blank">Astropy Paper II</a>
+                <li><a href="https://arxiv.org/abs/2206.14220" target="_blank">Astropy Paper III (v5.0)</a>
+                    (<a href="https://ui.adsabs.harvard.edu/abs/2022ApJ...935..167A" target="_blank">ADS</a> -
+                    <a href="https://ui.adsabs.harvard.edu/abs/2022ApJ...935..167A/exportcitation" target="_blank">BibTeX</a>)
+                </li>
+                <li><a href="https://arxiv.org/abs/1801.02634" target="_blank">Astropy Paper II (v2.0)</a>
                     (<a href="http://adsabs.harvard.edu/abs/2018arXiv180102634T" target="_blank">ADS</a> -
-                    <a href="http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2018arXiv180102634T&data_type=BIBTEX&db_key=PRE&nocookieset=1" target="_blank">BibTeX</a>)
+                    <a href="https://ui.adsabs.harvard.edu/abs/2018AJ....156..123A/exportcitation" target="_blank">BibTeX</a>)
                 </li>
-                <li><a href="https://doi.org/10.1051/0004-6361/201322068" target="_blank">Astropy Paper I</a>
+                <li><a href="https://doi.org/10.1051/0004-6361/201322068" target="_blank">Astropy Paper I (v0.2)</a>
                     (<a href="http://adsabs.harvard.edu/abs/2013A%26A...558A..33A" target="_blank">ADS</a> -
-                    <a href="http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2013A%26A...558A..33A&amp;data_type=BIBTEX&amp;db_key=AST&amp;nocookieset=1" target="_blank">BibTeX</a>)
+                    <a href="http://adsabs.harvard.edu/abs/2013A%26A...558A..33A/exportcitation" target="_blank">BibTeX</a>)
```

Comment:
Why not make all the URLs consistent?
```suggestion
                    <a href="https://ui.adsabs.harvard.edu/abs/2013A%26A...558A..33A/exportcitation" target="_blank">BibTeX</a>)
```

Reply:
Sure - I think this was just leftover from the old ADS interface